keyboards/handwired/boss566y/redragon_vara/keymaps/default/keymap.c

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2020 boss566y
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define GUI_APP     RGUI_T(KC_APP)
+#define C_HOME      RCTL(KC_HOME)
```

Comment:
Not sure if these make much sense, since you can use `G(KC_APP)` and `C(KC_HOME)` if you'd like shorter keycodes.

Reply:
I kbfirmware to make the board and and thus the those were just left as I didn't have any real reason to change them.
Also I don't think G(KC_APP) would not work for me since I need tap and window as separate items in the key. I can change them if necessary.